### PR TITLE
(doc) Fix typos in javadoc and comments

### PR DESCRIPTION
**Pull request by marcwrobel**
Fix minor typos in javadoc and comments.

### CODE REVIEW

File: commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/bzip2/Bzip2FileSystem.java [hunk 1]
```diff
@@ -27,7 +27,7 @@
 import org.apache.commons.vfs2.provider.compressed.CompressedFileFileSystem;
 
 /**
- * Filesytem to handle compressed files using the bzip2 method.
+ * Filesystem to handle compressed files using the bzip2 method.
```
Comment (garydgregory) [inline commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/bzip2/Bzip2FileSystem.java:30]: This is not right, it should either be "file system" or "FileSystem".

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/FileExtensionSelectorTest.java [hunk 2]
```diff
@@ -107,7 +107,7 @@ public void testManyExtensions() throws Exception {
                 Arrays.asList(foArray).toString());
         assertFalse(extensionSet.isEmpty(), message);
         assertEquals(ExtensionCount, extensionSet.size(), message);
-        // check all unique extensions
+        // check all uniques extensions
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/FileExtensionSelectorTest.java:110]: Not right.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java [hunk 3]
```diff
@@ -404,7 +404,7 @@ public void testRootFileName() throws Exception {
         // Test that the root path is "/"
         assertEquals("root path", "/", rootName.getPath());
 
-        // Test that the root basname is ""
+        // Test that the root basename is ""
```
Comment (garydgregory) [inline commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java:407]: "basename" is not a word either, it should be "base name".

**Review by garydgregory — CHANGES_REQUESTED** [review]
@marcwrobel 
Thank you for your PR, please see my comments.

File: commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/bzip2/Bzip2FileSystem.java (hunk 1 above)
Comment (marcwrobel) [reply commons-vfs2/src/main/java/org/apache/commons/vfs2/provider/bzip2/Bzip2FileSystem.java]: Done.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java [hunk 4]
```diff
@@ -188,7 +188,7 @@ public void testAbsoluteNames() throws Exception {
         name = getReadFolder().getFileSystem().getRoot().getName();
         checkAbsoluteNames(name);
 
-        // Test against some unknown file
+        // Test against some unknown files
```
[the remark this answers is not in the document]
Comment (marcwrobel) [reply commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java]: Done.

File: commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java (hunk 3 above)
Comment (marcwrobel) [reply commons-vfs2/src/test/java/org/apache/commons/vfs2/NamingTests.java]: Done.

**Pull request merged**